custom/icds_reports/ucr/reports/ls/ls_v2_ag_out_of_school.json

```diff
@@ -0,0 +1,142 @@
+{
+  "domains": [
+    "icds-dashboard-qa",
+    "sankalp_cas",
+    "zohaib-sandbox",
+    "akshita-sandbox",
+    "sunaina-sandbox",
+    "laurence-project-1",
+    "jessica-icds-cas",
+    "marissa-test",
+    "derek-icds-sandbox",
+    "priyanka-app",
+    "shrena-dev",
+    "aparatest",
+    "reach-sandbox",
+    "reach-dashboard-qa",
+    "reach-test",
+    "icds-test",
+    "icds-sql",
+    "icds-cas",
+    "cas-lab",
+    "icds-cas-sandbox"
+  ],
+  "server_environment": [
+    "india",
+    "icds"
+  ],
+  "report_id": "ls_v2_ag_out_of_school",
+  "data_source_table": "static-person_cases_v3",
+  "config": {
+    "title": "UCR V2 - LS - AG Out Of School",
+    "description": "",
+    "visible": false,
+    "aggregation_columns": [
+      "owner_id",
+      "month"
+    ],
+    "filters": [
+      {
+        "display": "Date Case Modified",
+        "slug": "modified_on",
+        "type": "date",
+        "field": "modified_on",
+        "datatype": "date",
+        "compare_as_string": false
+      },
+      {
+        "display": "Date of Birth",
+        "slug": "dob",
+        "type": "date",
+        "field": "dob",
+        "datatype": "date",
+        "compare_as_string": false
+      },
+      {
+        "display": "Filter by AWW",
+        "slug": "awc_id",
+        "type": "dynamic_choice_list",
+        "field": "awc_id",
+        "choice_provider": {
+          "type": "location"
+        },
+        "ancestor_expression": {
+          "field": "supervisor_id",
+          "location_type": "supervisor"
+        },
+        "datatype": "string",
+        "show_all": true
+      },
+      {
+        "display": "Filter by Supervisor",
+        "slug": "supervisor_id",
+        "type": "dynamic_choice_list",
+        "field": "supervisor_id",
+        "choice_provider": {
+          "type": "location"
+        },
+        "datatype": "string",
+        "show_all": true,
+        "ancestor_expression": {}
+      },
+      {
+        "required": false,
+        "type": "numeric",
+        "slug": "migration_status",
+        "field": "migration_status",
+        "display": "Whether Migrated",
+        "datatype": "integer"
+      },
+      {
+        "required": false,
+        "type": "numeric",
+        "slug": "registered_status",
+        "field": "registered_status",
+        "display": "Whether Availing Services",
+        "datatype": "integer"
+      }
+    ],
+    "columns": [
+      {
+        "display": "Month",
+        "column_id": "month",
+        "type": "aggregate_date",
+        "field": "modified_on",
```

Comment:
If it is possible for an AG case to be updated without the related person case being update this will be very confusing, since it will only show modified date for the person case.

Reply:
in this particular UCR the only indicator we want to fetch is is_oos from the person case itself. Basically trying to calculate in which month someone was out of school (is_oos = 1). This should not be interpreted in conjunction with when the ag_care case was modified which maybe whenever a service is delivered etc.

Reply:
I am not totally sure I understand what you are saying? This date will change every time the person case is modified (whether not oos is modified) and will not change if the person case is not modified (for example if the AG case is modified without modifying the person case). Notably, this won't show who was out of school in a given month, or even who was marked out of school in a given month, just whether or not the person case changed, for OOS or any other reason. If a case that was modified in April, but marked oos in February, it will appear show April. It sounds a little from that description that this column doesn't do exactly what you are intending. If it is what you are looking for, it might be a good idea to change the label of the column or add some kind of help text if possible, that makes it clear what it is showing (last modified date, not the month in which they were out of school). Changing the text to "last modified date" would also make it consistent with the filter, so that users now that the field there corresponds directly to the one they are filtering on.

Reply:
Hey @calellowitz , the idea behind aggregating on modified_date and not on open_date was that someone can be marked out of school separate from when the case was opened, but with the example yp mentioned I can see that modified date may also be confusing as it will continue to show the case for a month even when is_oos particularly wasn't updated for that case. 
Do you think an aggregation on open_date of the ag_care case and not person_case makes sense here? when we are trying to fetch AGs marked out of school in a month?

Reply:
Hey @orangejenny , I discussed this offline with @calellowitz and as mentioned over Skype, having modified_date in the person case DS would be useless because we won't be able to rebuild on TCL. We will be fetching out of school from the existing ag_care UCR itself. There fore, we should revert this addition of modified_date to the cases-person v3 (static) DS and can also go ahead and delete the new AG Out Of School UCR. This isn't urgent and won't need an adhoc deploy but can be merged with the weekly planned deploy on Monday on TCL and India.
Apologies for the back and forth here and thanks to both of you for your inputs and efforts!

Reply:
Making these changes in https://github.com/dimagi/commcare-hq/pull/27425

Reply:
Confirming these changes have been merged and deployed to TCL.

Reply:
They've also been deployed to india.